scripts/transforms/README.md

```diff
@@ -0,0 +1,17 @@
+## Codemods
+
+These codemods should be ran with `jscodeshift`
+
+### addWidget / removeWidget
+
+These should be replaced to addWidgets, removeWidgets
+
+```
+yarn jscodeshift -t scripts/transforms/addWidget-addWidgets.js --extensions='ts,js,tsx' examples stories .storybook
+```
+
+Make sure to run prettier after!
+
+```
+yarn prettier --write '{examples,stories,.storybook}/**/*.{js,ts,tsx}'
```

Comment:
Perhaps we can add a `format` script in the project.

Reply:
Can be useful